production/dea_ml/docker/requirements.txt

```diff
@@ -0,0 +1,115 @@
+# misc
+aiobotocore[boto3,awscli]==1.1.2
+docutils==0.15.2 # because of botocore
+pyyaml==5.3.1
+python-dateutil==2.8.0  # pinned because of pysal==2.2.0
+colorama==0.4.3
+cython
+ruamel.yaml
+ruamel.yaml.clib
+pyrsistent
+param
+dill
+ffmpeg-python
+tqdm
+voluptuous
+pydash
+s3fs
+# pre-install misc datacube dependencies
+jsonschema
+ciso8601
+psycopg2
+sqlalchemy
+toolz
+lark-parser
+setuptools_scm[toml]
+bottleneck
+# pre-install misc odc-tools dependencies
+lmdb
+zstandard
+thredds-crawler
+astropy
+boltons
+dataclasses
+# pre-install eodatasets3 dependencies
+cattrs
+structlog
+
+# Scientific Stack
+dask[complete]==2.30.0
+distributed==2.30.0
+dask-image
+dask-ml
+tensorflow==2.3.1
+scipy
+xgboost       # used by Dale's s2cloudmask
```

Comment:
delete

Reply:
dask-ml is used by predict_xr
scipy could be the dependency of scikit-learn

Reply:
Yes, I only intended for xgboost to be deleted, for some reason github shows a larger preview than the lines selected